phalcon/http/response.zep

```diff
@@ -14,6 +14,7 @@
  +------------------------------------------------------------------------+
  | Authors: Andres Gutierrez <andres@phalconphp.com>                      |
  |          Eduar Carvajal <eduar@phalconphp.com>                         |
+ |          Zamrony P. Juhara <zamronypj@yahoo.com>                       |
```

Comment:
Basically andres and eduar are added as authors always beacause they are authors of frameworks and you can you as author if you are **creating** new file, but othey than that authors should stay the same.